src/components/PanelCard/Section/PanelSection.styles.tsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+import { Section } from "@/app/globalStyles";
+import { Icon } from "@/components/Icon/Icon";
+import styled from "styled-components";
+
+export const Wrapper = styled(Section)`
+  margin: 2rem 0;
+`;
+
+export const Header = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 100%;
+`;
+
+export const RightIcon = styled(Icon)`
+  transform: rotate(-90deg);
+`;
+
+export const Title = styled.p`
+  font-size: 1.5rem;
+  font-weight: bold;
+  transition: 300ms;
+`;
+
+export const Subtitle = styled.p`
+  margin: 0;
+  max-width: 80rem;
+
+  @media screen and (max-width: 600px) {
+    text-align: justify;
+    text-align-last: center;
+  }
+`;
+
+export const PanelsContainer = styled.div`
+  display: flex;
+  column-count: 4;
+  flex-wrap: wrap;
+  width: 100%;
+  justify-content: center;
+  padding: 1rem;
+  align-items: center;
+  gap: 1.5rem;
+
+  @media screen and (max-width: 1000px) {
+    column-count: 2;
+    gap: 2rem;
+  }
+
+  @media screen and (max-width: 600px) {
+    column-count: 1;
+  }
```

Comment:
This suggestion make the grid more responsive

```suggestion
  display: grid;
  flex-wrap: wrap;
  width: 100%;
  justify-content: start;
  padding: 1rem;
  align-items: center;
  gap: 1.5rem;

  grid-template-columns: repeat(4, minmax(20rem, 1fr));

  @media screen and (max-width: 1200px) {
    grid-template-columns: repeat(3, minmax(200px, 1fr));
  }

  @media screen and (max-width: 700px) {
    grid-template-columns: repeat(2, minmax(200px, 1fr));
  }

  @media screen and (max-width: 400px) {
    grid-template-columns: repeat(1, minmax(200px, 1fr));
  }
```